ui/components/app/account-menu/index.scss

```diff
@@ -10,7 +10,9 @@
   color: var(--white);
 
   @media screen and (max-width: $break-small) {
-    right: calc(((100vw - 100%) / 2) + 8px);
+    position: initial;
+    margin-top: -10px;
+    margin-left: calc(((100vw - 100%) / 2) + 8px);
```

Comment:
Do we want to apply the same strategy for subsequent media queries for the sake of consistency?

Reply:
@darkwing I think this is actually better applied to only this media-query and leave all the others as `position:fixed` with the position explicitly set.

Otherwise in the full screen view we run into cases like:
<img width="494" alt="Screenshot 2022-01-12 at 16 56 21" src="https://user-images.githubusercontent.com/5368363/149175295-d82775d4-63e1-4650-911d-cbe2eaf8b805.png">

(This is obviously a very edge case and most likely no one shrinkes his browser window that much but still)